voidcrew/modules/more_pacmen/code/pacmen_machines.dm

```diff
@@ -0,0 +1,21 @@
+// Add your pacmen here name, circuit path, icon, power_gen, max_sheets, sheet path and time_per_sheet
+
+/obj/machinery/power/port_gen/pacman/voidman
```

Comment:
we should have the basic uranium/plasma/diamond ones too. The base TG one should be overwritten and we should re-include stock part effects because power is scarce here when most ships do not have a supermatter or SMES.

Reply:
I think the main pacemen code take care of the stock part effects it seems to

Reply:
Is that how it works currently? I think they just swap between using little and using a lot, and don't use stock parts